lib/SimplePie/SimplePie.php

```diff
@@ -1322,12 +1322,21 @@ public function enable_exceptions($enable = true)
 
 	function cleanMd5($rss)
 	{
-		return md5(preg_replace(array(
-			'#<(lastBuildDate|pubDate|updated|feedDate|dc:date|slash:comments)>[^<]+</\\1>#',
-			'#<(media:starRating|media:statistics) [^/<>]+/>#',
-			'#<!--.+?-->#s',
-			), '', $rss));
-		
+		$data = '';
```

Comment:
Please also try to submit the fix upstream SimplePie. :-) https://github.com/simplepie/simplepie (After @Alkarex comments.)

Also note the comments we've added to things that aren't upstream.

https://github.com/FreshRSS/FreshRSS/blob/98d7aef49264dea528f89df8b198e4bdc03cd5f1/lib/SimplePie/SimplePie.php#L1514-L1521

Reply:
This part to allow caching is not (yet) in SimplePie upstream, so this patch cannot directly be sent upstream

Reply:
No problem then, or is there already a PR for it?